plugins/TopPairElectronPlusJetsSelectionFilter.cc

```diff
@@ -324,9 +324,13 @@ bool TopPairElectronPlusJetsSelectionFilter::passesLooseElectronVeto() const {
 			}
 		}
 		return isZEvent == true;
-	} else
-		// Require only one loose electron, which is the signal electron
+	} 
+	else {
+		if (nonIsolatedElectronSelection_ || invertedConversionSelection_){
```

Comment:
The idea here is that the QCD electron will never be in `vetoElectrons`?
Will there not be a region of isolation where this is not true (e.g. for non-iso QCD region)?
I assume the conversion veto is applied for `vetoElectrons` as well.